Package_TO_SOT_SMD.pretty/Vishay_PowerPAK_SC70-6L_Single.kicad_mod

```diff
@@ -0,0 +1,43 @@
+(module Package_TO_SOT_SMD:Vishay_PowerPAK_SC70-6L_Single (layer F.Cu) (tedit 5AE7F3BE)
```

Comment:
Is it possible you used the export footprint feature to save these footprints?

This would explain why your footprint has the lib name included here.

(That is also why travis complains about the value field. It assumes the module name to be correct and does not handle cases that include the library name.)